scripts/README.md

```diff
@@ -36,19 +38,57 @@ The following steps must be proposed as a pull request.
 5. Open PR with changes and wait for review and green CI
 6. Merge PR once CI is green and it has been approved
 
-### Release
+#### Release
 
 1. Checkout the last version of the release branch
 2. `pnpm install && pnpm release:build` (make sure you have the latest dependencies installed, and build the packages)
 3. `pnpm release:publish` (release the versions on npm, you need your 2FA device)
 4. `pnpm release:tag` (push the newly created tag)
 
-### Documentation
+#### Documentation
 
 `pnpm docs:deploy` to deploy the documentation (it lives at https://material-ui.netlify.app/) with the latest changes.
 Force push if necessary.
 
-### Announce
+#### Announce
+
+After the docs is live, follow the instructions in https://mui-org.notion.site/Releases-7490ef9581b4447ebdbf86b13164272d.
+
+### Releasing a patch version
+
+A patch release goes like this:
+
+#### Prepare
+
+Checkout the latest minor release tag and create a branch "release/PATCH_VERSION". Cherry-pick the necessary commit on this branch. The following steps must be proposed as a pull request.
+
+1. Generate the changelog with `pnpm release:changelog`
+   The output must be prepended to the top level `CHANGELOG.md`
+   `pnpm release:changelog --help` for more information. If your GitHub token is not in your env, pass it as `--githubToken <my-token>` to the above command.
+
+2. Clean the generated changelog:
+   1. Match the format of https://github.com/mui/material-ui/releases.
+   2. Change the packages names casing to be lowercase if applicable
+3. Update the root `/package.json`'s version
+4. Run `pnpm release:version`. Keep in mind:
+   1. Only packages that have changes since the last release should have their version bumped.
+   2. If they have changes, packages that follow Material-UI's versioning scheme should be bumped to the same version as the root `package.json`. This might require skipping some version numbers.
+5. Open PR with changes and wait for review and green CI
+6. Merge PR once CI is green and it has been approved
+
+#### Release
```

Comment:
Are the version and changelog synced back to master, mainly for the latest one (v7)?
For example, if master is on 7.1.1 and in our normal release, we want to do 7.2.0. But we had to release 7.1.2 for some critical fix.
So next time we do 7.2.0, the changelog will jump directly from 7.1.1 to 7.2.0 if we follow the usual release process.